src/test/java/net/sf/jabref/logic/groups/GroupTreeNodeTest.java

```diff
@@ -14,15 +16,23 @@
 import org.junit.Test;
 
 import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
 
 public class GroupTreeNodeTest {
 
     private List<BibEntry> entries = new ArrayList<>();
+    private BibEntry entry;
 
     @Before
     public void setUp() throws Exception {
+        entries.clear();
+        entry = new BibEntry();
+        entries.add(entry);
         entries.add(new BibEntry().withField("author", "author1 and author2"));
         entries.add(new BibEntry().withField("author", "author1"));
+
+        Globals.prefs = JabRefPreferences.getInstance();
```

Comment:
Shouldn't the preferences setting go into a `@BeforeClass`-annotated method? Or do you need this for every test?